docs/content/sips/005-manifest-redesign.md

```diff
@@ -97,30 +131,17 @@ is equivalent to:
 base = "/"
 [[trigger.http]]
 route = "/hello"
-component = "hello"
-[[component]]
-id = "hello"
+handler = "hello"
+[component.hello]
+source = "hello.wasm"
 ```
 
-## Design options
-
-### Don't update the manifest version
-
-Instead of updating the manifest version we could not do that. It would be nice to include
-error message guidance and/or a `spin doctor` tool to upgrade the manifest format. This
-would be easy to detect if we switch the `spin_version` key.
+This upgrade could be performed automatically by `spin doctor`.
 
-### Trigger IDs
-
-We might want to be able to reference individual triggers, for example to implement a
-`spin up --trigger=world`. It would save some compatibility headache if we require IDs now:
+## Design options
```

Comment:
I think another change that might mitigate future confusion would be to rename `[component.config]` to `[component.variables]`. Especially if/when preview2 value imports and templates (which may be too far out) become a thing.

Reply:
Oh yeah I'll add that.